db/00000000000007_task_aggregation_counters.up.sql

```diff
@@ -0,0 +1,14 @@
+-- Per-task report aggregation counters, used for metrics.
+--
+-- Fillfactor is lowered to improve the likelihood of heap-only tuple optimizations. See the
+-- discussion around this setting for the task_upload_counters table.
+CREATE TABLE task_aggregation_counters(
+    id BIGINT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,  -- artificial ID, internal only
+    task_id BIGINT NOT NULL,  -- task ID the counter is associated with
+    ord BIGINT NOT NULL,  -- the ordinal index of the task aggregation counter
+
+    success BIGINT NOT NULL DEFAULT 0,  -- reports successfully aggregated
```

Comment:
Do we not want to track preparation failures by reason the way we do upload failures? If nothing else, keeping track of either total failures or total prep attempts would let us compute success rate.

Reply:
See response to inahga's comment:

> I'm implementing the least amount of functionality that gives us what we need; I'm OK leaning on operational metrics (e.g. prometheus/grafana) & logging for other issues, at least until we have a justification for splitting them out. (And yeah, I think it would be easy to add more to this later.)

Success rate, specifically, can be tracked via the operational metrics.